packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -784,20 +785,32 @@ contract Deploy is Deployer {
 
     /// @notice Deploy the SystemConfig
     function deploySystemConfig() public broadcast returns (address addr_) {
-        console.log("Deploying SystemConfig implementation");
-        SystemConfig config = new SystemConfig{ salt: _implSalt() }();
+        if (cfg.useInterop()) {
+            console.log("Deploying SystemConfigInterop implementation");
 
-        save("SystemConfig", address(config));
-        console.log("SystemConfig deployed at %s", address(config));
+            SystemConfigInterop config = new SystemConfigInterop{ salt: _implSalt() }();
+
+            addr_ = address(config);
+
+            save("SystemConfig", addr_);
+            console.log("SystemConfigInterop deployed at %s", addr_);
+        } else {
+            console.log("Deploying SystemConfig implementation");
+
+            SystemConfig config = new SystemConfig{ salt: _implSalt() }();
+
+            addr_ = address(config);
+
+            save("SystemConfig", addr_);
```

Comment:
You can dedupe `save` and `console.log` as they can be the same in both branches, we have a log that lets us know interop is happening already so no need to change `console.log`

Reply:
for sure! forgot to push a change here removing the branch